packages/price-exporter/src/utils.js

```diff
@@ -0,0 +1,46 @@
+/* @flow */
+
+import { flattenDeep } from 'lodash'
+
+// This method is necessary because of the custom fields, it is not possible
+// to know the actual headers on the fly so we would have to prepare the
+// headers before we start writing to CSV file
+export function _getHeaders (prices: Array<Object>) {
+  const headers = new Set()
+  prices.forEach((price) => {
+    if (price)
```

Comment:
looking to the signature `prices: Array<Object>`, to my understanding this would never be falsy. 
What's the reason for the if-statement?